test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/IntegrationTests.cs

```diff
@@ -40,7 +40,8 @@ public void ExportResultIsSuccess()
 
             var exporterOptions = new OtlpExporterOptions
             {
-                Endpoint = new System.Uri($"http://{CollectorEndpoint}"),
+                // TODO: Should test both http and https
+                Endpoint = new System.Uri($"https://{CollectorEndpoint}"),
```

Comment:
Just to vet out https support, I just changed the test to use https. I plan to follow up and have it test both.

Reply:
If we use a self-signed cert might need to manually pass validation somehow?

Reply:
> If we use a self-signed cert might need to manually pass validation somehow?

The cert gets installed as a trusted cert here, so no need to manually pass validation:

https://github.com/open-telemetry/opentelemetry-dotnet/blob/21bed212272d2260ce54f784f853cce9ced839e3/test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/Dockerfile#L18-L20

I had actually tried this first:

```csharp
ServicePointManager.ServerCertificateValidationCallback = (sender, certificate, chain, policyErrors) => true;
```

For some reason this did not work... I've used this kind of thing in test applications before, but for some reason the callback was not getting invoked by the test.

Reply:
`ServicePointManager.ServerCertificateValidationCallback` doesn't work for the Google library or .NET or both? I'm not too surprised if it doesn't work in the Google library. .NET would be more of a surprise 😄